english/marshmallow.md

```diff
@@ -21,7 +21,7 @@ Grave danger. Oh No!
 I actually used to know a guy named Dave Granger. We sure had some good times! Anyway, the possibility of danger lurks.
 
 The type of extreme mortal danger that so often accompanies confectionary inspired interior design.
-Quickly surveying your immediate surrounds, you notice a garishly dressed, 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.